WooCommerce/Classes/ViewRelated/Orders/OrderListViewModel.swift

```diff
@@ -351,6 +351,34 @@ final class OrderListViewModel {
     }
 }
 
+// MARK: - In-Person Payments Feedback Banner
+
+extension OrderListViewModel {
+    func dismissIPPFeedbackBanner(remindAfterDays: Int?) {
+        //  Updates the IPP feedback banner status as dismissed
+        let updateFeedbackStatus = AppSettingsAction.updateFeedbackStatus(type: .IPP, status: .dismissed) { [weak self] _ in
+            self?.hideIPPFeedbackBanner = true
+        }
+        stores.dispatch(updateFeedbackStatus)
+
+        //  Updates the IPP feedback banner status to be reminded later, or never
+        let updateBannerVisibility = AppSettingsAction.setFeatureAnnouncementDismissed(
+            campaign: .IPP,
+            remindAfterDays: remindAfterDays,
+            onCompletion: nil
+        )
+        stores.dispatch(updateBannerVisibility)
+    }
+
+    func remindMeLaterTapped() {
```

Comment:
We can name this function more specifically for the IPP Feedback Banner.

Reply:
Renamed on https://github.com/woocommerce/woocommerce-ios/pull/8657/commits/9884f22a964bcc50e635af902154bd427e2798da